### PR TITLE
 Graph US Gov endpoints old v/s new updates

### PR DESCRIPTION
**Pull request by negoe**
Update page https://docs.microsoft.com/en-us/graph/deployments 
Reason: To provide clarity on using  graph.microsoft.com and graph.microsoft.us. graph.microsoft.com is going to be obsolete and customer should use the new endpoint.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b2770c3](https://github.com/negoe/microsoft-graph-docs/pull/4245/commits/b2770c352e10629d4107bc19957191de2da708ac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/deployments.md](https://github.com/negoe/microsoft-graph-docs/blob/USGovGraph/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903252015155622-4245%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ahU9DGS1FHaHnqznFkwh7acIX31Xq7ADqcjAp5pTk7s%3D&st=2019-03-25T20%3A14%3A12Z&se=2019-04-25T20%3A19%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/deployments.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ The following table shows the service root endpoints for Microsoft Graph and [Gr
 | Microsoft Graph China operated by 21Vianet | https://microsoftgraph.chinacloudapi.cn | https://developer.microsoft.com/zh-cn/graph/graph-explorer-china |
 | Microsoft Graph global service | https://graph.microsoft.com | https://developer.microsoft.com/graph/graph-explorer |
 
+>**Important:** If you already have an app in US Government and you're using the `https://graph.microsoft.com` endpoint, we recommend that you use the new `https://graph.microsoft.us` endpoint. The `https://graph.microsoft.com` endpoint will continue to work but will eventually be obsolete. Please keep an eye on this page for the timeline.
+
```
Comment (Lauragra) [inline concepts/deployments.md]: I suggest removing the "Please keep an eye on..." sentence. This change should be announced via the Changelog and a blog post. I don't think it's reasonable to expect that a user will continue to return to a page to look for updates.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6c00c49](https://github.com/negoe/microsoft-graph-docs/pull/4245/commits/6c00c49f139979f10c674325e3d789ac0d0d3eb8)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/deployments.md](https://github.com/negoe/microsoft-graph-docs/blob/USGovGraph/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903252326021016-4245%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=nyykDdDfeuk3i1i9z8yeDnSRLP8S7Cx4zpu4PfpC7rQ%3D&st=2019-03-25T23%3A24%3A25Z&se=2019-04-25T23%3A29%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[b296c4c](https://github.com/negoe/microsoft-graph-docs/pull/4245/commits/b296c4c5e53f4c7cfd97871a732d7843d13cf3c6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/deployments.md](https://github.com/negoe/microsoft-graph-docs/blob/USGovGraph/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C1%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904012302253061-4245%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=bz%2Bp6IH6p8VRGv5mzTdoaRxDnkOB3uQ8uIheXQWzujI%3D&st=2019-04-01T23%3A00%3A38Z&se=2019-05-02T23%3A05%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by negoe** [comment]
@Lauragra

**Comment by jthake-msft** [comment]
Docs Build status updates of commit _[3faecf4](https://github.com/negoe/microsoft-graph-docs/pull/4245/commits/3faecf4becc0141c760dc0b31d16802dd186b77f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[concepts/deployments.md](https://github.com/negoe/microsoft-graph-docs/blob/USGovGraph/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201904020500327911-4245%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SZEFsLjMDukJMJr8z2HzdeCU2W3SU7INoCtsllohF3Q%3D&st=2019-04-02T04%3A58%3A54Z&se=2019-05-03T05%3A03%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**